### PR TITLE
Tweak positioning of the file picker

### PR DESCRIPTION
**Pull request by elliette**
Makes the file picker wider, and moves it a little to the left.

Screen Shot 2021-10-01 at 3 19 11 PM

### CODE REVIEW

File: packages/devtools_app/lib/src/debugger/codeview.dart [hunk 1]
```diff
@@ -402,15 +402,18 @@ class _CodeViewState extends State<CodeView>
     );
   }
 
-  Widget wrapInElevatedCard(Widget widget) {
+  Widget wrapInElevatedCard(
+    Widget widget, {
+    double width = wideSearchTextWidth,
```
Comment (jacob314) [inline packages/devtools_app/lib/src/debugger/codeview.dart:407]: this extra parameter is never passed in anywhere so should be removed.

File: packages/devtools_app/lib/src/debugger/codeview.dart [hunk 2]
```diff
@@ -423,6 +426,7 @@ class _CodeViewState extends State<CodeView>
       FileSearchField(
         debuggerController: widget.controller,
       ),
+      width: extraWideSearchTextWidth,
```
Comment (jacob314) [inline packages/devtools_app/lib/src/debugger/codeview.dart:429]: what effect does specifying this width and the width for the container achieve?

File: packages/devtools_app/lib/src/debugger/codeview.dart (hunk 1 above)
Comment (elliette) [reply packages/devtools_app/lib/src/debugger/codeview.dart:407]: Which extra parameter? The `width` parameter is being used on line 429 below

File: packages/devtools_app/lib/src/debugger/codeview.dart (hunk 2 above)
Comment (elliette) [reply packages/devtools_app/lib/src/debugger/codeview.dart:429]: Hmm I'm confused - this width is the width for the container (in`wrapInElevatedCard`)

**Review by kenzieschmoll — APPROVED** [review]

File: packages/devtools_app/lib/src/debugger/codeview.dart (hunk 2 above)
Comment (jacob314) [reply packages/devtools_app/lib/src/debugger/codeview.dart:429]: What confuses me is its child also has a width (line 416). I'd hope you could just specify one width for the outer parent and have all widths be computed based on that.

Comment (kenzieschmoll) [reply packages/devtools_app/lib/src/debugger/codeview.dart:429]: you may need to expand out the few lines that are omitted in the git diff. This line (429) is part of the `buildFileSearchField` method, which calls `wrapInElevatedCard` with the parameter `width: extraWideSearchTextWidth,`.  The above use of `width` on line 416, is part of the `wrapInElevatedCard` method.

**Pull request merged**